MapboxCoreNavigation/RouteController.swift

```diff
@@ -262,6 +262,20 @@ open class RouteController: NSObject {
      */
     var rawLocation: CLLocation?
     
+    public var reroutingTolerance: CLLocationDistance {
+        guard let intersections = routeProgress.currentLegProgress.currentStepProgress.intersectionsIncludingUpcomingManeuverIntersection else { return RouteControllerMaximumDistanceBeforeRecalculating }
+        guard let userLocation = rawLocation else { return RouteControllerMaximumDistanceBeforeRecalculating }
+        
+        for intersection in intersections {
+            let distanceToIntersection = distance(along: routeProgress.currentLegProgress.currentStepProgress.step.coordinates!, from: userLocation.coordinate, to: intersection.location)
```

Comment:
If we’re concerned about performance, I think we can use as-the-crow-flies distance instead of the distance along the route line, since anything significantly far from the route line would result in rerouting anyways. Sounds like circular logic, though – needs verification.

Reply:
I was thinking similarly.`distance(along:)` sounds overly expensive here.

Reply:
As the crow flies sounds good to me.